recipes/send-message/index.ts

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv';
+dotenv.config({ path: '../.env' });
+
+import WhispirClient from '../../api';
+
+const client = WhispirClient({
+    host: 'https://stage-ap-southeast-2.whispirdev.com/api',
+    username: process.env.WHISPIR_USERNAME,
+    password: process.env.WHISPIR_PASSWORD,
+    apiKey: process.env.API_KEY,
+});
+
+const workspaceId = process.env.WORKSPACE_ID || '';
+
+async function main() {
+    const requestParams = {
```

Comment:
Could we firstly create a contact, in this recipe, then use the contact created via `client.contacts.create` and send a message to them? I think that better demonstrates the interaction between the `contacts` and `messages` resource.